.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Some text show up on the same line when previewed as markdown. Is that intentional? (I don't have admin access and can't preview as issue template in settings)